### PR TITLE
Changes to API

### PR DESCRIPTION
**Pull request by n1mda**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -927,7 +922,7 @@
                       <strong>Accept</strong>
                     </div>
                     <div class="col-md-9">
-                      <span class="value">application/vnd.budbee.multiple.orders-v1+json</span>
+                      <span class="value">application/vnd.budbee.multiple.orders-<strong>v2</strong>+json</span>
```
Comment (heldtalex) [inline index.html]: Why are some v2 in bold and others not?

File: index.html [hunk 2]
```diff
@@ -1372,100 +1307,114 @@
...
+}</code></pre>
+
+                    </div>
+                  </div>
+
+                </div>
+              </div>
+            </div>
+
+            <div id="update-orders-address--resources" class="subgroup">
```
Comment (heldtalex) [inline index.html]: We should group "Create an Order", "Get Orders", "Update an Order" and "Cancel an Order" under one menu option. Also make sure we're consistent with "capital O" Order, "create order" vs "update **an** order".

File: index.html (hunk 1 above)
Comment (n1mda) [inline index.html]: For clarity. But come to think about it, @nicke will all endpoints work with v2?

File: index.html [hunk 3]
```diff
@@ -1288,76 +1286,13 @@
...
-    "address": {
-      "street": "Grevgatan 10",
-      "street2": "",
-      "postalCode": "11453",
-      "city": "Stockholm",
-      "country": "SE"
-    },
     "additionalInfo": "",
-    "doorCode": "1212",
+    "doorCode": "<strong>1212</strong>",
```
Comment (heldtalex) [inline index.html]: This styling will not work.

File: index.html (hunk 1 above)
Comment (heldtalex) [inline index.html]: I would say it's less clear when some are bold and some not. Either all are in bold or none are.

Comment (nicksen) [inline index.html]: I've only added it for the "new" create order endpoint. How should it work?

**Comment by heldtalex** [comment]
This depends on https://github.com/sendus/budbee-php/pull/1

**Pull request merged**